### DISCUSSION TITLE
Multi API documentation support

### DISCUSSION THREAD

**Discussion by neoplomero** [DarkaOnLine/L5-Swagger#375] [General]
Hi, 

I just saw that on the new releases annotations. I'd like to know if you could point some place where I can read some documentation about how to implement this Multi API documentation. Thanks.

**Comment by ymhuang0808** [comment]
I also have the same question about multiple API documentation.

**Reply by neoplomero** [reply]
Hi, I realized that if you want to use multiple documentation you need multiple sources of notations. Check the config file and you'll see how to set the different sources of notations and different URLs for the destination. A bit disappointing for my case, I was expecting to be able to have multiple docus based on the same notations maybe using tags or so. If I have the time I'll try to collaborate with something like this.
